test/testdata/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -4074,7 +4074,6 @@ items:
   metadata:
     annotations:
       authorization.openshift.io/system-only: "true"
-      rbac.authorization.kubernetes.io/autoupdate: "true"
```

Comment:
This does not seem right - the default bootstrap policy should always have this set to true.